draft-harrison-sshm-mlkem.md

```diff
@@ -80,17 +79,17 @@ informative:
 
 --- abstract
 
-This document defines Post-Quantum key exchange methods based on Module-lattice post-quantum key encapsulation schemes.  These methods are defined for use in the SSH Transport Layer Protocol.
+This document defines pure post-quantum key exchange methods based on Module-lattice post-quantum key encapsulation schemes for use in the SSH Transport Layer Protocol.
 
 --- middle
 
 # Introduction
 
-Secure Shell (SSH) [RFC4251] is a secure remote login protocol. The key exchange protocol described in [RFC4253] supports an extensible set of methods. The security of traditional key exchange methods used in Secure Shell (SSH) [RFC4251] relies on the algorithms being too computationally complex to be broken. The development of quantum computers poses a threat to the complexity of these algorithms. Given sufficiently powerful quantum computers, these traditional algorithms would be vulnerable to attack.
+Secure Shell (SSH) [RFC4251] is a secure remote login protocol. The key exchange protocol described in [RFC4253] supports an extensible set of methods. The security of traditional key exchange methods used in Secure Shell (SSH) [RFC4251] relies on the algorithms being too computationally complex to be broken. The development of quantum computers poses a threat to the complexity of these algorithms. Given sufficiently powerful quantum computers, these traditional algorithms would be vulnerable to attack. Additionally, the threat of "harvest-now-decrypt-later" attacks could creates a risk in the current landscape before sufficiently powerful quantum computers are available. In this attack, the data would be collected and decrypted by these quantum computers at a later date.
 
-Additionally, the threat of "harvest-now-decrypt-later" attacks could creates a risk in the current landscape before sufficiently powerful quantum computers are available. In this attack, the data would be collected and decrypted by these quantum computers at a later date.
+This document addresses the problem by proposing the use of post-quantum key encapsulation mechanisms (KEMs) to extend the SSH [RFC4253] key exchange. [I-D.ietf-sshm-ntruprime-ssh] introduces ML-KEM in PQ/T Hybrid mode [draft-ietf-pquip-pqt-hybrid-terminology] which combines the shared secrets established by an ECDH and a ML-KEM key exchange. This document uses ML-KEM in a single-algorithm scheme without combining it with a traditional ECDH exchange. 
```

Comment:
Sorry, wrong reference here. It should be [draft-ietf-sshm-mlkem-hybrid-kex] instead of the NTRU one